lib/cli/commands/versions.js

```diff
@@ -0,0 +1,43 @@
+// Versions
+const baseMiddleware = require("../middlewares/base.js");
+const requireg = require("requireg");
+
+let versions = {
+	command: "versions",
+	describe: "Show versions of all UI5 tooling modules",
+	middlewares: [baseMiddleware]
+};
+
+const getVersion = (pkg) => {
+	const NOT_FOUND = "===(not installed)";
+	try {
+		const packageInfo = requireg(`${pkg}/package.json`);
+		return packageInfo.version || NOT_FOUND;
+	} catch (err) {
+		return NOT_FOUND;
+	}
+};
+
+versions.handler = (argv) => {
+	try {
+		const cliVersion = getVersion("@ui5/cli");
```

Comment:
As `@ui5/cli` is *this* module, I guess we could also use `../../..`? This would then skip most of the resolution logic and ensure that always *this* module is being used.

Reply:
That's a good suggestion! I will update it.